app/scss/main.scss

```diff
@@ -26,6 +25,27 @@
   }
 }
 
+main {
+  & #welcome-section {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(700px, 1fr));
+
+    & .welcome-section-left {
+      padding: 100px 0px 50px 150px;
+      font-size: 46px;
+      color: #7452a3;
+    }
+    & .welcome-section-right {
+      font-size: 200px;
+      display: grid;
```

Comment:
for this propose you can remove display: grid; justify-items: end;
align-items: end; and use display: block; margin-left: auto;